### PR TITLE
registering for job challenge

### PR DESCRIPTION
**Pull request by konnster99**
Hey maintainer! please accept my PR.

### CODE REVIEW

File: content/participants/ttjc@konnster99/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+I need an internship in JUNE 2020.
+I want to learn.
+
+## something interesting about you?
+
+Well I like like to watch movies and vibe with the punjabi music.
+
+### help I need?
+
+I would be needing a lot of help because I literslly will be learning js formthe scratch in this 1 month.
```
Comment (khusharth) [inline content/participants/ttjc@konnster99/index.md:28]: * I literslly
* fromthe

File: content/participants/ttjc@konnster99/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+### how to contact me?
+
+- Twitter:'https://twitter.com/nanokoder'
```
Comment (khusharth) [inline content/participants/ttjc@konnster99/index.md:32]: Please use the markdown format to add all your links like: 
```
[Twitter](https://twitter.com/nanokoder)
```

**Comment by konnster99** [comment]
Sir could you please re-check, sorry for the errors.

**Comment by konnster99** [comment]
sir what does dikkat hi dikkat mean

**Comment by konnster99** [comment]
do i have to restart the registering process??

File: content/participants/ttjc@konnster99/index.md [hunk 3]
```diff
@@ -21,20 +21,20 @@ I want to learn.
 
 ## something interesting about you?
 
-Well I like like to watch movies and vibe with the punjabi music.
+Well I like to watch movies and vibe with the punjabi music.
 
 ### help I need?
 
-I would be needing a lot of help because I literslly will be learning js formthe scratch in this 1 month.
+I would be needing a lot of help because I literally will be learning js form the scratch in this 1 month.
```
Comment (khusharth) [inline content/participants/ttjc@konnster99/index.md:28]: from* scratch

**Comment by khusharth** [comment]
No you don't need to register again. Just fix the things where you made a mistake and your PR will be merged.

File: content/participants/ttjc@konnster99/index.md (hunk 3 above)
Comment (konnster99) [reply content/participants/ttjc@konnster99/index.md:28]: oohk i'm really sorry for the typos.
Will update

**Comment by konnster99** [comment]
Could you just recheck??

File: content/participants/ttjc@konnster99/index.md [hunk 4]
```diff
@@ -25,7 +25,7 @@ Well I like to watch movies and vibe with punjabi/bollywood music.
 
 ### help I need?
 
-I would be needing a lot of help because I literally will be learning js form the scratch in this 1 month.
+I would be needing a lot of help because I literally will be learning js form scratch in this 1 month. So I'll just ping in the tanay pratap group if I have any issues.
```
Comment (khusharth) [inline content/participants/ttjc@konnster99/index.md:28]: form -.> **from**

Comment (konnster99) [reply content/participants/ttjc@konnster99/index.md]: resolved

**Comment by khusharth** [comment]
@gkanishk Good to merge.

**Pull request merged**